### PR TITLE
Latest vm common v1.0.0

### PR DESCRIPTION
**Pull request by miiu96**
Use the latest release on the [elrond-vm-common](https://github.com/ElrondNetwork/elrond-vm-common/tree/v1.0.0) repository.
In this new release, all the` fungible ESDT operations `will generate a log with details about the operation that is executed.

Fixed mandos tests.

### CODE REVIEW

File: test/egld-esdt-swap/mandos/wrap_egld.scen.json [hunk 1]
```diff
@@ -39,6 +39,14 @@
                 "status": "0",
                 "message": "",
                 "logs": [
+                    {
+                        "address": "sc:egld-esdt-swap",
+                        "identifier": "str:ESDTTransfer",
+                        "topics": [
+                            "str:WEGLD-abcdef", "500",  "address:user"
```
Comment (camilbancioiu) [inline test/egld-esdt-swap/mandos/wrap_egld.scen.json:46]: Format list as one element per line, for readability and consistency with the other Mandos tests.

Comment (miiu96) [reply test/egld-esdt-swap/mandos/wrap_egld.scen.json]: done.

File: go.mod [hunk 2]
```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/ElrondNetwork/big-int-util v0.1.0
 	github.com/ElrondNetwork/elrond-go-logger v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v0.3.4-0.20210707100510-6cef7cea933f
+	github.com/ElrondNetwork/elrond-vm-common v1.0.0
```
Comment (sasurobert) [inline go.mod:8]: you hae to referentiate arwen v1_3 at line 21 with the latest commit hash

Comment (miiu96) [reply go.mod:8]: done.

**Review by sasurobert — APPROVED** [review]

**Review by camilbancioiu — APPROVED** [review]

**Pull request merged**